### DISCUSSION TITLE
blog/graphql-over-internet

### DISCUSSION THREAD

**Discussion by giscus** [the-guild-org/the-guild-website#1184] [Docs Discussions]
# blog/graphql-over-internet

HTTP, WebSockets, Server-Sent Events, undergoing standardization and libraries.

https://the-guild.dev/blog/graphql-over-internet

**Comment by deathemperor** [comment]
https://github.com/graphql/graphql-over-http/blob/main/rfcs/GraphQLOverSSE is missing .md extension => 404
